src/app/checkboxes/checkboxes.demo.html

```diff
@@ -61,9 +61,51 @@
                 </div>
             </div>
         </div>
+        <div class="form-group">
+            <label>Native Indeterminate Checkbox</label>
+            <div class="checkbox-inline">
+                <input type="checkbox"
+                       id="checkrads_10"
+                       [indeterminate]="nativeIndeterminateState">
+                <label for="checkrads_10">A native indeterminate checkbox.</label>
+            </div>
+        </div>
+    </section>
+</form>
+
+<form #form="ngForm" novalidate>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With ngForm & ngModel</label>
+            <div class="checkbox-inline">
+                <clr-checkbox
+                        name="termsCheckbox"
+                        id="checkrads_11"
+                        [clrChecked]="termsAgreement"
+                        [ngModel]="termsAgreement">
+                    I agree to the terms and conditions.
+                </clr-checkbox>
+            </div>
+            <div class="checkbox-inline">
+                {{ form.value | json }}
+            </div>
+        </div>
     </section>
 </form>
 
+<form>
+    <section class="form-block">
+        <div class="form-group">
+            <label>With a list of objects</label>
+            <div class="checkbox" *ngFor="let item of list">
```

Comment:
Same here. I thought the `.checkbox` class is added to the host automatically. I do see it on the demo app:
![image](https://cloud.githubusercontent.com/assets/1426805/22109963/d7bbc55e-de0e-11e6-8b23-d73cbcc540ae.png)

I am not sure why we have to wrap it again in `.checkbox`